gdal/frmts/fits/fitsdataset.cpp

```diff
@@ -67,6 +87,11 @@ class FITSDataset : public GDALPamDataset {
                               int nXSize, int nYSize, int nBands,
                               GDALDataType eType,
                               char** papszParmList );
+  virtual const char *GetProjectionRef();
```

Comment:
GetPojectionRef() / SetProjection() are no longer virtual method in latest master after https://trac.osgeo.org/gdal/wiki/rfc73_proj6_wkt2_srsbarn . See MIGRATION_GUIDE.TXT

Reply:
Sorry, I did read the rfc, but clearly not fully understood. I think that 0adbc25a00811cc29c7741270bb39e113a49e399 fixes the issue.